orion/pkg/router/handler.go

```diff
@@ -72,6 +71,7 @@ func (h *Handler) SetupApiEndpoints() {
 		apiSessions.GET("/v1/session/:id", controllers.GetSessionById)
 		apiSessions.POST("/v1/session/:id", controllers.UpdateSession)
 		apiSessions.DELETE("/v1/session/:id", controllers.DeleteSession)
+		apiClasses.GET("/v1/class/:classId", controllers.GetAllSessionsByClassId)
```

Comment:
I think I meant in this line `/v1/sessions/class/:classId`

Reply:
OH nvm I see it. You're using apiClasses here

Reply:
Oh I'm silly. My bad.